ci/gpu/build.sh

```diff
@@ -60,7 +60,7 @@ source activate gdf
 conda install "cudf=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
                "dask-cudf=$MINOR_VERSION.*" "dask-cuda=$MINOR_VERSION.*" \
                "rapids-build-env=$MINOR_VERSION.*" \
-               "rapids-notebook-env=$MINOR_VERSION.*"
+               "rapids-notebook-env=$MINOR_VERSION.*" "cugraph=$MINOR_VERSION.*"
```

Comment:
Minor thing, but can we add `cugraph` in alphabetical order?

Reply:
Sure